patches/@stencil+react-output-target+0.7.1.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/node_modules/@stencil/react-output-target/dist/react-output-target.cjs b/node_modules/@stencil/react-output-target/dist/react-output-target.cjs
+index 295a892..a296484 100644
+--- a/node_modules/@stencil/react-output-target/dist/react-output-target.cjs
++++ b/node_modules/@stencil/react-output-target/dist/react-output-target.cjs
+@@ -13,12 +13,12 @@
+ import React from 'react';
+ import { ${f} } from '@stencil/react-output-target/runtime';
+ import type { EventName, StencilReactComponent } from '@stencil/react-output-target/runtime';
+-  `);for(const p of n){const u=p.tagName,l=T(u),$=`${l}Element`,b=`${l}CustomEvent`;e.addImportDeclaration({moduleSpecifier:`${s}/${i}/${u}.js`,namedImports:[{name:l,alias:$},{name:"defineCustomElement",alias:`define${l}`}]});const F=(p.events||[]).filter(o=>o.internal===!1),y=[];for(const o of F)if(Object.keys(o.complexType.references).length>0){for(const _ of Object.keys(o.complexType.references))o.complexType.references[_].location==="global"||e.addImportDeclaration({moduleSpecifier:s,namedImports:[{name:_,isTypeOnly:!0}]});e.addImportDeclaration({moduleSpecifier:s,namedImports:[{name:b,isTypeOnly:!0}]}),y.push({originalName:o.name,name:j(o.name),type:`EventName<${b}<${o.complexType.original}>>`})}else y.push({originalName:o.name,name:j(o.name),type:`EventName<CustomEvent<${o.complexType.original}>>`});const E=`${l}Events`;e.addTypeAlias({name:E,type:y.length>0?`{ ${y.map(o=>`${o.name}: ${o.type}`).join(`,
++  `);for(const p of n){const u=p.tagName,l=T(u),$=`${l}Element`,b=`${l}CustomEvent`;e.addImportDeclaration({moduleSpecifier:`${s}/${i}/${u}.js`,namedImports:[{name:l,alias:$},{name:"defineCustomElement",alias:`define${l}`}]});const F=(p.events||[]).filter(o=>o.internal===!1),y=[];for(const o of F)if(Object.keys(o.complexType.references).length>0){for(const _ of Object.keys(o.complexType.references))o.complexType.references[_].location==="global"||e.addImportDeclaration({moduleSpecifier:s,namedImports:[{name:_,isTypeOnly:!0}]});e.addImportDeclaration({moduleSpecifier:s,namedImports:[{name:b,isTypeOnly:!0}]}),y.push({originalName:o.name,name:j(o.name),type:`EventName<${b}<${o.complexType.original}>>`})}else y.push({originalName:o.name,name:j(o.name),type:`EventName<CustomEvent<${o.complexType.original}>>`});const E=`${l}Events`;e.addTypeAlias({name:E,type:y.length>0?`{ ${y.map(o=>`'${o.name}': ${o.type}`).join(`,
```

Comment:
Leaving traces about this patch. This patch is to enable event names with '/' in them like "atomic/redirect". This change is only 4 lines. 2 lines in both .cjs and .js file.

Precisely this : 
<img width="291" alt="image" src="https://github.com/user-attachments/assets/1946c875-8d46-474a-ba06-3233bff56edc">


And this :

<img width="517" alt="image" src="https://github.com/user-attachments/assets/71439874-d9bb-4396-b367-23d8b8cf7962">